docs/data/ingest-library/examples/ledger_entry_statistics.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Get ledger entries from History Archive
+sidebar_position: 20
+---
+
+This example connects to the Stellar History Archive using `historyarchive.Connect` and initializes a checkpoint reader with `ingest.NewCheckpointChangeReader` to read ledger entry changes at a given checkpoint. It iterates over the ledger changes, processing different entry types such as `Account`, `ClaimableBalance`, `Trustline`, and `Offer`, using the `ingest.Change` structure. The code tracks statistics in an `entriesInfo` struct, counting the total entries and specific types.
+
+<CodeExample>
+
+```go
+// Filename: example_ledger_entries.go
+
+package main
+
+import (
+    "context"
+    "encoding/json"
+    "fmt"
+    "github.com/stellar/go/network"
+    "io"
+    "time"
+
+    "github.com/stellar/go/historyarchive"
+    "github.com/stellar/go/ingest"
+    "github.com/stellar/go/support/storage"
+    "github.com/stellar/go/xdr"
+)
+
+func panicIf(err error) {
+    if err != nil {
+        panic(fmt.Errorf("An error occurred, panicking: %s\n", err))
+    }
+}
+
+type entriesInfo struct {
+    TotalEntries              int32
+    NumAccounts               int32
+    NumAccountsWithTrustlines int32
+
+    NumClaimableBalances int32
+    NumOffers            int32
+}
+
+func ledgerEntryStats() {
+    archiveURLs := network.TestNetworkhistoryArchiveURLs
+    networkPassphrase := network.TestNetworkPassphrase
+
+    // Open a history archive using our existing configuration details.
+    historyArchive, err := historyarchive.Connect(
```

Comment:
Using [ArchivePool](https://github.com/stellar/go/blob/master/historyarchive/archive_pool.go#L35) would be best convention when configuring from a list of urls, probably less code as well, such as:

```
historyArchive, err := NewArchivePool(archiveURLs, ArchiveOptions{})
```